src/workloads/scale/BigUpdate.yml

```diff
@@ -129,8 +129,10 @@ AutoRun:
       $eq:
       - atlas
       - replica
-      - replica-noflowcontrol
       - replica-1dayhistory-15gbwtcache
       - replica-all-feature-flags
+      - replica-noflowcontrol
       - single-replica
       - standalone
+      - standalone-classic-query-engine
```

Comment:
Is there a reason that we want "BigUpdate" running on the standalone variants and not the 1 node repl set variants? looking at https://evergreen.mongodb.com/task/sys_perf_linux_1_node_replSet_big_update_5cbe7f207d1abe694708bf729e4808a4a5413e60_22_09_28_21_01_35, this seems possible, but I'm not sure if this is deliberate.